itests/hive-unit/src/test/java/org/apache/hadoop/hive/ql/parse/TestReplicationScenariosAcidTables.java

```diff
@@ -1645,15 +1644,20 @@ public void testHdfsMaxDirItemsLimitDuringIncremental() throws Throwable {
     WarehouseInstance.Tuple incrementalDump2 = primary.run("use " + primaryDbName)
             .dump(primaryDbName, dumpClause);
 
-    eventCount = Integer.parseInt(incrementalDump2.lastReplicationId)
-            - Integer.parseInt(incrementalDump1.lastReplicationId);
-    assertTrue(eventCount > 5);
+    eventCount = getNoOfEventsDumped(incrementalDump2.dumpLocation);
+    assertTrue(eventCount > 5 && eventCount < 1000);
```

Comment:
What is the significance of 5 and 1000 here?

Reply:
This test is to assert that no of events dumped cannot be greater than (DFS_MAX_DIR_ITEMS_CONFIG - RESERVED_DIR_ITEMS_COUNT). For this particular run of repl dump (line no. 1645), This value is 1000 which was initially 5 for the previous run. So, no of events should be greater than 5 and less than 1000.